oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/tagtranslator/OSMTagInterface.java

```diff
@@ -1,9 +1,10 @@
 package org.heigit.ohsome.oshdb.util.tagtranslator;
 
-/*
- * This interface allows to write methods that accept both OSMTag and OSMTagKey
+/**
+ * An OSM tag ({@link OSMTag}) or an OSM tag key ({@link OSMTagKey}).
+ *
+ * <p>This interface allows to write methods that accept both OSMTag and OSMTagKey
  * objects as parameters without overloading, and it allows to write methods that
- * return either an OSMTag or an OSMTagKey object.
+ * return either an OSMTag or an OSMTagKey object.</p>
```

Comment:
The closing `</p>` is actually not necessary, just to this here

Reply:
yeah, I find it personally slightly more pleasing-to-the-eye/symetrical to have also the closing `<p>` tags even though they are not strictly necessary as you rightly say. :nerd_face: